### ISSUE TITLE
Faster sampling

### ISSUE THREAD

**Issue by rosgori** [fonnesbeck/gp_regression#2]
For faster sampling

``` python
import os
# This must happen before pymc is imported, so you might
# need to restart the kernel for it to take effect.
os.environ['MKL_NUM_THREADS'] = '1'
os.environ['OPENBLAS_NUM_THREADS'] = '1'
```

When you use binder, it takes a lot of time.

**Comment by fonnesbeck** [comment]
Thanks, I will add these when I get a chance.

**Comment by fonnesbeck** [comment]
Closed by 003ff005ca13890bb370593dd5139350d2c3fff9
